### DISCUSSION TITLE
Sense resistor value

### DISCUSSION THREAD

**Discussion by indistinctTalk** [bassamanator/Sovol-SV06-firmware#159] [Q&A]
Hi, just curious how this value was determined?

https://github.com/bassamanator/Sovol-SV06-firmware/blob/b48ad8a8078418a9e9046b16da63cc42643d9cfd/printer.cfg#L43

I'm guessing borrowed from here? 

https://github.com/Sovol3d/Sv06-Source-Code/blob/ba1b9e11815d3202d5fd7871e35c18f7416b4ab1/Marlin/Configuration_adv.h#L2596

**Comment by bassamanator** [comment]
All values pulled from official Marlin source code, there might be exceptions, however.
